react-locate-contractor/src/Pages/BusinessPage.tsx

```diff
@@ -1,12 +1,149 @@
-import React from "react";
+import { useState, useEffect } from 'react';
+import { useOktaAuth } from '@okta/okta-react';
+import Button from '@mui/material/Button';
+import { Typography, Card, CardContent, CardActions, Container } from '@mui/material';
+function BusinessPage() {
+    <h3 id="personal">Business</h3>
+    const { authState, oktaAuth } = useOktaAuth();
+    const [userInfo, setUserInfo]: [userInfo: any, setUserInfo: any] = useState(null);
+    const businessName = "TODO: A Business Name";
+    const specialty = "TODO: a specialty"
+    const aboutBusiness = "TODO: A very good business";
+    const serviceOffered = "TODO: These are my services offered";
+    const hoursOperation = "TODO: Hours of operation";
+    const yearsBusiness = "TODO: A lot of years";
+    const phone = "TODO: 612-321-3211";
+    const email = "TODO: todo@gmail.com";
 
-const BusinessPage = () => {
-    return (
+    useEffect(() => {
+        if (!authState || !authState.isAuthenticated) {
+            // When user isn't authenticated, forget any user info
+            setUserInfo(null);
+        } else {
+            oktaAuth.getUser().then((info: any) => {
+                setUserInfo(info);
+            }).catch((err) => {
+                console.error(err);
+            });
+        }
+    }, [authState, oktaAuth]); // Update if authState changes
+
+    if (!userInfo) {
+        return (
+            <div>
+                <p>Uh oh, you are not logged in! To access you're business account you must be logged in...</p>
+            </div>
+        );
+    }
 
+    return (
         <div>
-            <h1>Business</h1>
-        </div>
+            <div>
+                <Typography variant="button" id="welcome">
+                    Welcome, &nbsp;{userInfo.name}!
+                </Typography >
+                <Container>
+                    <Card variant="outlined" sx={{ maxWidth: 500, height: 140 }} >
```

Comment:
Looks like a lot of repeatable code here when building out the cards. Can this instead all be summed up into one single reusable component?